README.md

```diff
@@ -189,6 +189,25 @@ us-central1-docker.pkg.dev/genuine-flight-317411/devel/krr:v1.8.3
 We do **not** recommend installing KRR from source in airgapped environments due to the headache of installing Python dependencies. Use one of the above methods instead and contact us (via Slack, GitHub issues, or email) if you need assistance.
 </details>
 
+<details>
+  <summary>In-Cluster</summary>
+
+Apart from running KRR as a CLI tool you can also run KRR inside your cluster. We suggest installing KRR via the Robusta platform. It gives you a free UI with some features like the following
+* View application usage history graphs on which recommendations are based.
+* Get application, namespace and cluster level recommendations.
+* YAML configuration to apply the suggesetd recommendation and more
+
+You can also run KRR in-cluster as a Kubernetss Job, if you don't want to view results easily in a UI. 
+
+Create all the necessary Kubernetes services using the following command to run KRR as a Kubernetes Job.
```

Comment:
I think you can leave this line out. It's obvious from the line above what the following command does.